src/utility/Ex4.py

```diff
@@ -0,0 +1,40 @@
+import numpy as np
+import scipy.stats as stats
+import json
+jsonfile=open('data\input.json','r')
+readfile=jsonfile.read()
+data=json.loads(readfile)
+ward1=data["wardDistribution"].get("distribution")
+ward=[]
+for each in ward1:
+    ward.append(each)
+ward.pop()
+#doc du lieu triple tu Pedestrian.json
+triple=0
+n=data["numOfAgents"].get("value")
+jsonfile2=open('data\Pedestrian.json','r')
+readfile2=jsonfile2.read()
+data2=json.loads(readfile2)
+for each in data2:
+    if each["type"]=="Personel":
+        triple+=1
+single=n-triple
+print(triple)
+#sinh du lieu
+max_value=n-len(ward)*3
+min_value=1
+mean=(max_value+min_value)/2
```

Comment:
I think we should set `mean = (triple*3+simple)/len(ward)` in order to force the value of `sum(sample)`, rather than retry until it works.